content/concepts/introduction-to-webpack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introduction
+---
+
+*Webpack* is a module system for modern Javascript applications.
+
+In [modular programming](https://en.wikipedia.org/wiki/Modular_programming), developers break programs up into discrete chunks of functionality.
```

Comment:
I feel, we must first explain modules and then talk about modular programming. Like "A module is code that contains a discrete chunk of functionality. Developers use this technique to break their programs into smaller, maintainable modules. This is called modular programming(link)."